### PR TITLE
feat(dns): dns managed zone and dns record

### PR DESCRIPTION
**Pull request by bluebunrabs**

### CODE REVIEW

File: dns.tf [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  name        = "iamjustintime-livelinklabs-delegation"
+  dns_name    = "${var.dns_name}."
+  description = "Delegated zone for iamjustintime application"
+}
+
+resource "google_dns_record_set" "default" {
+  project = var.project
+  name = google_dns_managed_zone.default.dns_name
+  type = "A"
+  ttl  = 300
```
Comment (Tensho) [inline dns.tf]: I guess 24 hours is OK for us 👌 We always can downscale it to 1 min if there will be a need to re-create Cloud Run deployment. Remember, we pay for each DNS request in the cloud.
```suggestion
  ttl  = 86400
```

**Review by Tensho — APPROVED** [review]

File: dns.tf [hunk 2]
```diff
@@ -0,0 +1,16 @@
+resource "google_dns_managed_zone" "default" {
+  project = var.project
+  name        = "iamjustintime-livelinklabs-delegation"
+  dns_name    = "${var.dns_name}."
```
Comment (new-light-106) [inline dns.tf:4]: Blockchainlog.in

Comment (new-light-106) [reply dns.tf:4]: Blockchainlog.in

**Comment by new-light-106** [comment]
Blockchainlog.in

**Review by new-light-106 — CHANGES_REQUESTED** [review]
Blockchainlog.in

**Pull request merged**